Tests/Tests UI/test_register_page.py

```diff
@@ -0,0 +1,34 @@
+from Tests.PageObject.RegisterPage import RegisterPage
+from Tests.PageObject.locators import BasePageLocators
+
+base_link = "http://parabank.parasoft.com"
+
+
```

Comment:
please add a fixture that would run before all tests and start the browser and execute commands

page = RegisterPage(browser, base_link)
    page.open()
        
So the browser would be loaded only once before all tests